### DISCUSSION TITLE
resource redis

### DISCUSSION THREAD

**Discussion by Luchi-Fer** [filamentphp/filament#8949] [General]
Can you create a resource to display data from a redis database?

**Comment by leandrocfe** [comment]
If you have a Laravel Model, you can use it on the resource (readonly)

**Reply by Luchi-Fer** [reply]
Thanks for answering. That is the problem I have, I don't have any model, only a repository class in which I have the function to save and another to obtain the data from redis

**Reply by leandrocfe** [reply]
Resources are static classes that are used to build CRUD interfaces for your **Eloquent models**

There is a workaround using [sushi](https://filamentphp.com/community/how-to-consume-an-external-api-with-filament-tables)

**Reply by Luchi-Fer** [reply]
perfect, thank you very much
